AsyncDisplayKitTests/ArrayDiffingTests.m

```diff
@@ -10,13 +10,58 @@
 
 #import "NSArray+Diffing.h"
 
+@interface NSArray (ArrayDiffingTests)
+- (NSIndexSet *)_asdk_commonIndexesWithArray:(NSArray *)array compareBlock:(BOOL (^)(id lhs, id rhs))comparison;
```

Comment:
Would recommend not writing test for `_asdk_commonIndexesWithArray:`, as it's a private method. Should only test the output of the public methods.